Realm/RLMAppCredentials.mm

```diff
@@ -55,6 +55,18 @@ + (instancetype)credentialsWithJWT:(NSString *)token {
     return [[self alloc] initWithAppCredentials: app::AppCredentials::custom(token.UTF8String)];
 }
 
++ (instancetype)credentialsWithFunction:(NSString *)payloadJson {
```

Comment:
We should not be exposing this as an NSString. It should be an NSDictionary.

Reply:
changed to NSDictionary